src/test/resources/regressions/issues/000282.gobra

```diff
@@ -0,0 +1,6 @@
+package issue282
```

Comment:
Our `.gobra` regressions generally must start with the following license, otherwise our CI complains
```
// Any copyright is dedicated to the Public Domain.
// http://creativecommons.org/publicdomain/zero/1.0/
```